Sources/Widgets/Widgets3D/ImageCroppingWidget/index.js

```diff
@@ -168,16 +166,19 @@ function vtkImageCroppingWidget(publicAPI, model) {
 
 // ----------------------------------------------------------------------------
 
-const DEFAULT_VALUES = {
-  // cornerManipulator: null,
-  // edgeManipulator: null,
-  // faceManipulator: null
-};
+const defaultValues = (initialValues) => ({
+  cornerManipulator: null,
+  edgeManipulator: null,
+  faceManipulator: null,
```

Comment:
when they are set to null, we typically comment then with @jourdain

Reply:
That's fine, so long as we don't strictly check these values for null.